tests/mainnet/state/validation/account_trie_node.yaml

```diff
@@ -8,10 +8,10 @@
-    content_value_retrieval: '0x04000000f85180a07f152c1e0fbe4b406b9a774b132347f174f02f3c2d6d1d4ad005c979996754b28080808080808080808080a06225fcc63b22b80301d9f2582014e450e91f9b329b7cc87ad16894722fff5296808080'
+- state_root: '0x1ad7b80af0c28bc1489513346d2706885be90abb07f23ca28e50482adb392d61'
```

Comment:
nit: I would add a comment that this the node that is parent of the leaf node from above.

You could be even more specific and repeat entire section:
```
# Block 19,000,000
# Address: 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2 (WETH)
# Branch node, corresponding to the path: 8679e8e (parent of the leaf node)
```

Also below and in the other file.